server/db_queries.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require("mongoose");
+const express = require("express");
+const router = express.Router();
+const conn_string =
+  "mongodb+srv://admin:passwordpassword@cluster0.wapen.mongodb.net/plusev?retryWrites=true&w=majority";
+
+//connect to our database which currently has 3 tables:
+//events/tournaments/markets
+mongoose.connect(conn_string, {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useUnifiedTopology: true,
+});
+
+mongoose.connection.on("connected", () => {
+  console.log("Successfully connected to the plusev database.");
+});
+
+mongoose.connection.on("error", (err) => {
+  console.error("Error connecting to mongodb", err);
+});
+
+//defining the schemas/models corresponding to the existing tables in the db
+
+const eventschema = new mongoose.Schema({});
+const tournamentschema = new mongoose.Schema({});
+const marketschema = new mongoose.Schema({});
+
+var events = mongoose.model("event", eventschema);
+var tournaments = mongoose.model("tournament", tournamentschema);
+var markets = mongoose.model("market", marketschema);
+
+router.get("/api/events", async (req, res, next) => {
```

Comment:
I "think" the way to do this would be to leave the actual routes in server.js and just have this file export a bunch of methods (or maybe a class or something) for manipulating the tables in the DB. That way any other parts of our server that need to use the DB would all use the same methods and we encapsulate all that table manipulation logic in here. It usually makes it easier to be consistent with how we use the DB, we can check/validate what's going into it here too then. 

Server.js can then also just use these methods in each of the routes. 

When the project is bigger we might have routes that are querying multiple tables and it would be good to have all queries to the DB happen in a consistent way without having to duplicate query logic. 

I'm fine with leaving it like this for now, just thought I'd mention it for later.